mirage/utils/constants.py

```diff
@@ -62,6 +62,16 @@
 # to when it is not in the beam
 NIRISS_GRISM_THROUGHPUT_FACTOR = 0.8
 
+# Lists of the NIRCam/FGS exposure order associated with each type of
```

Comment:
Note, typo in spelling of 'ORDER' ("OREDER"). 
A simplification is possible here, in that the order of exposures is always the same; just smaller subsets are executed depending on which flavor of GA observation. In particular, the "CORRECT+ADJUST" and "ADJUST2" types (which are identical) are the full set of 7. Everything else is just a truncation of that.